pkg/render/logstorage.go

```diff
@@ -1202,6 +1224,8 @@ func (es elasticsearchComponent) kibanaCR() *kbv1.Kibana {
 					EmptyDir: &corev1.EmptyDirVolumeSource{}}})
 	}
 
+	kibanaSecrets := append(es.kibanaSecrets, es.cfg.KibanaCertSecret)
```

Comment:
I don't think this is needed. A version of KibanaCertSecret is already in es.kibanaSecrets so there shouldn't be a need to add it again.

Reply:
Makes sense, the other version is a copy made alongside the component creation so it will always be up to date. Removed